source/features/comments-time-machine-links.tsx

```diff
@@ -91,10 +91,7 @@ async function showTimemachineBar(): Promise<void | false> {
 			return false;
 		}
 
-		const lastCommitDate = await elementReady([
-			'.repository-content .Box.Box--condensed relative-time',
-			'[itemprop="dateModified"] relative-time' // "Repository refresh" layout
-		].join(), {waitForChildren: false});
+		const lastCommitDate = await elementReady('[itemprop="dateModified"] relative-time', {waitForChildren: false});
```

Comment:
I messed this one up. I deleted the wrong one

Reply:
I see. I was seeing the bar because in Safari is still on 21.6.1

Reply:
I did attempt to fix it in the last PR... 

Whatever..